Makefile

```diff
@@ -43,8 +43,8 @@ docker-login:
 
 build: $(PLUGIN_DIR)
 	# load plugin with versionied tag
-	docker plugin rm -f ${PLUGIN_NAME} 2>/dev/null || true
-	docker plugin create ${PLUGIN_NAME} ./$(PLUGIN_DIR)
+	# docker plugin rm -f ${PLUGIN_NAME} 2>/dev/null || true
+	# docker plugin create ${PLUGIN_NAME} ./$(PLUGIN_DIR)
```

Comment:
Can we just remove these two lines?

Reply:
I think we may want to keep it as a reference, just like `make deploy`.